zgrab2/multiple.go

```diff
@@ -0,0 +1,31 @@
+package zgrab2
+
+import (
+	"errors"
+	"os"
+
+	"github.com/ajholland/zflags"
+)
+
+type MultipleConfig struct {
```

Comment:
Not convinced this struct is named well. A config file doesn't necessarily define multiple scans, does it? Why isn't this part of the other the regular config struct? Can the config file override the global config things like `GOMAXPROCS` or input/output file?